modules/administration-guide/partials/con_che-architecture-overview-with-dw.adoc

```diff
@@ -0,0 +1,19 @@
+[id="{prod-id-short}-architecture-overview-with-dw_{context}"]
+= Future {prod-short} architecture based on the DevWorkspace overview
+
+This section describes the future architecture of {prod-short} with the DevWorkspace {orch-name} Operator as workspace engine. {prod-short} default workspace engine is currently the {prod-id-short}-server. This will eventually change when the DevWorkspace will reach feature parity with {prod-id-short}-server. To configure {prod-id-short} to use the DevWorkspace follow the instructions in xref:enabling-dev-workspace-engine.adoc[this article]. 
+
+{prod} components are:
+
+* A central workspace controller: an always-running service that manages user workspaces through the {orch-name} API.
+* User workspaces: container-based IDEs that the controller stops when the user stops coding.
+
+.High-level {prod-short} architecture
+image::administration-guide:architecture/{project-context}-high-level.png[]
+
+When {prod-short} is installed on a {platforms-name} cluster, the workspace controller is the only component that is deployed. A {prod-short} workspace is created immediately after a user requests it.
+
+.Additional resources
+
+* xref:administration-guide:che-workspace-controller-with-dw.adoc[]
+* xref:administration-guide:che-workspaces-architecture-with-dw.adoc[]
```

Comment:
These pages don't exist. (See link checker errors).

Reply:
Was this caused by a fork from the old master branch?